tron/config/config_parse.py

```diff
@@ -593,6 +587,7 @@ class ValidateMesos(Validator):
     config_class = ConfigMesos
     option = True
     defaults = {
+        'master_address': None,
```

Comment:
We should have a validator defined for this below.

Reply:
Ah, right.